src/CIQS-CLI-Extension/azext_ciqs/format.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from collections import OrderedDict
+from collections import defaultdict
+
+def transform_templateListItem(result):
+    result = OrderedDict([('Template Id', result['id']),
+                        ('Title', result['title']),
+                        ('Category', result['category'])])
+    return result
+
+def transform_templateList(template_list):
+    return [transform_templateListItem(i) for i in template_list]
+
+def transform_templateView(result):
+    result = OrderedDict([('Title', result['title']),
+                        ('Category', result['category']),
+                        ('Estimated Time', result['estimatedTime']),
+                        ('Description', result['description'])])
+    return result
+
+STATUS_DICT = defaultdict(lambda: 'Unknown',
```

Comment:
This variable is not used.

Reply:
STATUS_DICT is used on line 35

Reply:
Ah my mistake. How about 'Created', 'ParameterSubmitted' and 'Interrupted'?

Consider putting these model specific constants in a separate directory/file.

Reply:
That sounds like a great idea.